vagrant.rb

```diff
@@ -90,15 +120,15 @@ def configure(config, os, pf_name, pf_id, host_name, host_id,
 
   # provisioning script
   if os == $windows7 or os == $windows2008 then
-    command = "c:/vagrant/scripts/network.cmd #{net} #{host_list}\n"
+    command = "c:/vagrant/scripts/network.cmd #{net} @host_list@\n"
```

Comment:
ok ! saw it!   command.sub("@host_list@", host_list) 

It's confusing with the host_list variable passed as parameter